spec/features/user_cannot_checkout_until_logged_in_spec.rb

```diff
@@ -16,7 +16,7 @@
 
     visit "/cart"
 
-    expect(page).to_not have_link("Checkout")
+    expect(page).to_not have_button("Checkout")
```

Comment:
Hooray for fixing tests before we merge https://media.giphy.com/media/11sBLVxNs7v6WA/giphy.gif